backend/de.metas.printing/de.metas.printing.base/src/main/java/de/metas/printing/api/impl/AbstractPrintingDAO.java

```diff
@@ -80,6 +84,8 @@ public abstract class AbstractPrintingDAO implements IPrintingDAO
 	 */
 	private static final ModelDynAttributeAccessor<I_C_Printing_Queue_Recipient, Boolean> DYNATTR_DisableAggregationKeyUpdate = new ModelDynAttributeAccessor<>("DisableAggregationKeyUpdate", Boolean.class);
 	private final IQueryBL queryBL = Services.get(IQueryBL.class);
+	private final WorkplaceService workplaceService = SpringContextHolder.instance.getBean(WorkplaceService.class);
```

Comment:
i think this might be a problem. what if the class is initialized before the spring-context is ready?
i think this service should either be injected via constructor or if that's no feasilbe, be not a field but a variable within the method(s) it is used in